fastlane/lanes/release.rb

```diff
@@ -466,7 +445,25 @@ def release_branch_name
   "release/#{release_version_current}"
 end
 
-def commit_version_bump
+def bump_build_codes
+  bump_production_build_code
+  bump_internal_build_code
```

Comment:
Might just be me, but I find it better and more readable to have two methods name "bump ..." than having the logic inline with a comment that explains what it does.

From ["Clean Code" by Uncle Bob](https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882):

<img width="500" alt="Screen Shot 2019-10-31 at 9 23 28 am" src="https://user-images.githubusercontent.com/1218433/67903657-4a477380-fbc0-11e9-8f6a-e3c3f210f92a.png">